app/translations/translate.py

```
@@ -159,18 +169,22 @@ def prioritize_existing_translations(po, untranslated_entries):
+        f"[prioritize_existing_translations] Prioritized {prioritized_count} existing translations"
+    )
 
 
 def translate(path, lang):
     try:
-        print(f"Processing {lang}: {path}")
+        logger.info(f"[main] Processing {lang}: {path}")
         po = polib.pofile(path)
     except Exception as e:
-        print(f"Error reading {path}: {e}")
+        logger.exception(f"[main] Error reading {path}: {e}")
```

Ruff (0.13.3)

Redundant exception object included in `logging.exception` call

(TRY401)